backend/contributions/views.py

```
@@ -1489,9 +1491,15 @@ def filter_min_accepted_contributions(self, queryset, name, value):
         return queryset
 
     def filter_has_proposal(self, queryset, name, value):
```

Ruff (0.15.20)

[warning] Unused method argument: `name`

(ARG002)